agent/tests/test_lib_aws_sdk_php.c

```diff
@@ -61,24 +61,83 @@ NR_PHP_WRAPPER(expect_arg_value_null) {
 }
 NR_PHP_WRAPPER_END
 
+NR_PHP_WRAPPER(aws_dynamodb_set_params_wrapper) {
+  (void)wraprec;
+  nr_segment_cloud_attrs_t cloud_attrs = {0};
+  nr_datastore_instance_t instance = {0};
+  nr_segment_datastore_params_t datastore_params = {0};
+  char* expected_string = NULL;
+  char* test_name_string = NULL;
+  zval* test_name = NULL;
+  zval* expected = NULL;
+
+  datastore_params.instance = &instance;
+
+  /*
+   * argument 1 is is used to pass in the test name
+   */
+
+  test_name = nr_php_get_user_func_arg(3, NR_EXECUTE_ORIG_ARGS);
```

Comment:
Is this supposed to take `1` as an argument instead of `3`?

Reply:
Good catch,
please see: [efd8799](https://github.com/newrelic/newrelic-php-agent/pull/1037/commits/efd87996476535e265a3d9b51053c23f518f4c20)